src/ДЗ 12 Typescript. Виведення типів.ts

```diff
@@ -0,0 +1,51 @@
+interface MatchFilter {
+  type: 'match';
```

Comment:
Just to let you know, using an enumeration for that field would be an ideal decision.